mne/beamformer/_dics.py

```diff
@@ -437,35 +992,97 @@ def tf_dics(epochs, forward, noise_csds, tmin, tmax, tstep, win_lengths,
-        bandwidth. Only used in 'multitaper' mode.
+        bandwidth. Only used in 'multitaper' mode. Defaults to True.
+    cwt_n_cycles: float | list of float | None
+        Number of cycles to use when constructing Morlet wavelets. Fixed number
+        or one per frequency. Defaults to 7.
+        Only used in 'cwt_morlet' mode.
+    decim : int | slice
+        To reduce memory usage, decimation factor during time-frequency
+        decomposition. Defaults to 1 (no decimation).
+        Only used in 'cwt_morlet' mode.
+
+        If `int`, uses tfr[..., ::decim].
+        If `slice`, uses tfr[..., decim].
+    subtract_evoked : bool
+        If True, subtract the averaged evoked response prior to computing the
+        tf source grid. Defaults to False.
     reg : float
-        The regularization for the cross-spectral density.
+        Regularization to use for the DICS beamformer computation.
+        Defaults to 0.05.
     label : Label | None
-        Restricts the solution to a given label.
-    pick_ori : None | 'normal'
-        If 'normal', rather than pooling the orientations by taking the norm,
-        only the radial component is kept.
+        Restricts the solution to a given label. Defaults to None.
+    pick_ori : None | 'normal' | 'max-power'
+        The source orientation to estimate source power for:
+
+            ``None`` :
+                orientations are pooled. (Default)
+            'normal' :
+                filters are computed for the orientation tangential to the
+                cortical surface
+            'max-power' :
+                filters are computer for the orientation that maximizes
+                spectral power.
+
+        Defaults to ``None``.
+    inversion : 'single' | 'matrix'
+        This determines how the beamformer deals with source spaces in "free"
+        orientation. Such source spaces define three orthogonal dipoles at each
+        source point. When ``inversion='single'``, each dipole is considered
+        as an individual source and the corresponding spatial filter is
+        computed for each dipole separately. When ``inversion='matrix'``, all
+        three dipoles at a source vertex are considered as a group and the
+        spatial filters are computed jointly using a matrix inversion. While
+        ``inversion='single'`` is more stable, ``inversion='matrix'`` is more
+        precise. See section 5 of [4]_.  Defaults to 'single'.
+    weight_norm : None | 'unit-noise-gain'
+        How to normalize the beamformer weights. None means no normalization is
+        performed.  If 'unit-noise-gain', the unit-noise gain minimum variance
+        beamformer will be computed (Borgiotti-Kaplan beamformer) [2]_.
+        Defaults to ``None``.
+    normalize_fwd : bool
+        Whether to normalize the forward solution. Defaults to ``True``. Note
+        that this normalization is not required when weight normalization
+        (``weight_norm``) is used.
```

Comment:
This presumably could be done independently for each orientation, or jointly across orientations for each location. I assume it's the latter but probably worth saying here or in `Notes`.

Reply:
It depends on the `inverse='single' | 'matrix'` parameter. With `inverse='single'`, this is done independently for each orientation. With `inverse='matrix'` it is done jointly. This makes the most sense to me: either we treat dipoles as separate sources, or we treat the groups of 3 dipoles as a single source.

Reply:
I still think this should (also) be controllable by the user directly. I.e., make your settings described above the default but also allow to set ``normalize_fwd = 'vertex'`` etc.

Reply:
I've tried it and my simple simulation results go haywire if I set `inversion='matrix', normalize_fwd='single'`. For now, I'm going to keep it to a simple `normalize_fwd=False|True`. We can always add more options later if we actually run into scenario's where we need them.